schemas/schema-generate.json

```diff
@@ -0,0 +1,279 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/SchemasMap",
+  "definitions": {
+    "SchemasMap": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/Schema"
+      }
+    },
+    "Schema": {
+      "$ref": "#/definitions/JSONSchema7"
+    },
+    "JSONSchema7": {
```

Comment:
Is it useful to have this be the output? What if it was just an array of output files or something like that?